lib/Frontend/CompilerInvocation.cpp

```diff
@@ -1420,7 +1420,14 @@ static bool ParseSILArgs(SILOptions &Opts, ArgList &Args,
   }
 
   if (const Arg *A = Args.getLastArg(options::OPT_sanitize_EQ)) {
-    Opts.Sanitize = parseSanitizerArgValues(A, Triple, Diags);
+    Opts.Sanitize = parseSanitizerArgValues(
+        A, Triple, Diags,
+        /* sanitizerRuntimeLibExists= */[&](const StringRef libName) {
+
+          // The driver has checked the existance of the library
```

Comment:
I believe so, it's the confusing two-stage check. If the library wasn't there, we would have errorred out by now.